file-tracker/file_tracker/client_connection.py

```diff
@@ -0,0 +1,51 @@
+import json
+import logging
+
+from aiortc import RTCPeerConnection, RTCSessionDescription
+from file_operations import ls, tail
+
+# STUN/TURN server configuration
+ICE_SERVERS = [{
```

Comment:
Please add: 
Interactive Connectivity Establishment (ICE)

and some link to external documentation

Reply:
we can use 
PROD: ice.inductiva.ai
DEV: ice-dev.inductiva.ai